package.json

```diff
@@ -16,34 +16,35 @@
     "ios",
     "cpp",
     "*.podspec",
-    "!lib/typescript/example",
-    "!android/build",
     "!ios/build",
+    "!android/build",
+    "!android/gradle",
+    "!android/gradlew",
+    "!android/gradlew.bat",
+    "!android/local.properties",
     "!**/__tests__",
     "!**/__fixtures__",
     "!**/__mocks__",
-    "!apps"
+    "!**/.*"
   ],
   "scripts": {
```

Comment:
I think we should probably add another script to fetch the latest submodule changes. thoughts? 

```
    "postinstall": "git submodule update --init --recursive && git submodule foreach git pull origin main"
```